lib/presets.js

```diff
@@ -428,6 +428,63 @@ module.exports = {
           return callback(error);
         }
       });
+    },
+
+    /**
+     * @summary Update quoted version immediately following a regex
+     * @function
+     * @public
+     *
+     * @param {Object} options - options
+     * @param {String} [options.baseDir] - directory to prefix onto options.file
+     * @param {String} options.file - file to modify
+     * @param {String} options.regex - regex leading up to the quoted version string
+     * @param {String} [options.regexFlags] - any modifier flags to apply to the regex
+     * @param {String} cwd - current working directory
+     * @param {String} version - version
+     * @param {Function} callback - callback (error)
+     * @returns {null}
+     *
+     * @example
+     * presets.updateVersion.quoted({file: "myfile.h", regex: /^VERSION\s+=\s+/}, process.cwd(), '1.0.0', (error) => {
```

Comment:
1. Using `\s*` instead of `\s+` could be a better choice here?
2. Should `regexFlags: 'm'` be used in conjunction with `^`?
3. Single quotes for `file: "myfile.h"` -> `file: 'myfile.h'`

Reply:
All very good points :+1:

Reply:
...except if I change it to `/^VERSION\s*=\s*/` then the interpreter sees the `*/` as the end of the comment-block, so I've switched it back to `/^VERSION\s+=\s+/` :wink: